### ISSUE TITLE
Fast model switching (UI/UX)

### ISSUE THREAD

**Issue by skt-t1-byungi** [egoist/chatkit-docs#5]
I want to be able to switch between models more quickly.
Currently, it requires multiple mouse clicks and searching. I want faster model switching.
(Model favorites or keyboard support for autocomplete suggestions come to mind)

**Comment by egoist** [comment]
makes sense, i'm gonna add a command-palette (cmd+k) similar to vscode's for fast model switch and other stuff with just keyboard
